setup/action.yaml

```diff
@@ -90,7 +90,7 @@ runs:
       if: steps.target-config.outputs.aws_assume_role_arn != ''
       with:
         role-to-assume: ${{ steps.target-config.outputs.aws_assume_role_arn }}
-        role-session-name: samplerolesession
+        role-session-name: ${{ steps.target-config.outputs.aws_role_session_name }}
```

Comment:
If role-session-name is empty, to be use `GitHubActions` automatically.
https://github.com/aws-actions/configure-aws-credentials/blob/2cefa29f8797029f898b1baeff3e21a144128687/src/index.ts#L34
https://github.com/aws-actions/configure-aws-credentials/blob/2cefa29f8797029f898b1baeff3e21a144128687/src/index.ts#L16